src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$colors:
+  "1" rgb(255, 0, 0),
+  "2" rgb(204, 0, 0),
+  "3" rgb(153, 0, 0),
+  "4" rgb(102, 0, 0),
+  "5" rgb(51, 0, 0),
+  "6" rgb(0, 0, 0);
```

Comment:
this could be done with scss loop

Reply:
it is used in loop @each, or what do you mean?